site/docs/education/egeria-dojo/developer/overview.md

```diff
@@ -29,7 +29,7 @@ This dojo was last tested using Egeria release 3.12.
 
     !!! attention "Think about your machine type ..."
 
-        The instructions below cover fairly standard machine architectures and set ups.  If you are using, say the new M1 chip in your Mac, or are running a work machine that has been locked down by your company, then you may need to take additional actions, install additional software and enable additional permissions. In all cases, check that the software being installed is correct for your machine type and its approved uses.
+        The instructions below cover fairly standard machine architectures and set ups.  If you are using, say the new M1 chip in your Mac, or Windows, or are running a work machine that has been locked down by your company, then you may need to take additional actions, install additional software and enable additional permissions. In all cases, check that the software being installed is correct for your machine type and its approved uses.
```

Comment:
I suspect we can remove m1 reference now as enough of the team are using m1 just fine (the only real issue we had was with architecture specific binaries -- that was only containers).

The issue with windows remains